tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -631,7 +631,7 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
       )
       _ = assertNoDiff(artefactCompletionList, artefactExpectedCompletionList)
 
-      versionExpectedCompletionList = List("0.14.1", "0.14.0")
+      versionExpectedCompletionList = List("0.14.2", "0.14.1", "0.14.0")
```

Comment:
Maybe instead of using `0.14` version for tests we could use one that is no longer published?

Reply:
Good idea, I will change it in all tests with `CoursierComplete` so these bumps won't be needed